### PR TITLE
build(deps): bump yarn from 3.2.3 to 3.4.1

### PR DESCRIPTION
**Pull request by yeikel**
I do not know what version we are targeting. Can we upgrade to the latest `3.x` version if CI passes? 

If the intention is to stay within `3.2.x`, I can create a separate PR  as we're also behind in that range

### CODE REVIEW

File: npm_and_yarn/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM ghcr.io/dependabot/dependabot-updater-core
 ENV DEPENDABOT_HOME /home/dependabot
 ### JAVASCRIPT
+ARG YARN_VERSION=3.5.0
```
Comment (jeffwidman) [inline npm_and_yarn/Dockerfile:4]: ❤️

**Review by jeffwidman — COMMENTED** [review]
Nice cleanup!

I don't know about the version bump, is it a breaking change compared to `yarn` `3.2`, or is it fully backwards compatible?

@pavera you may have some insights here?

**Comment by yeikel** [comment]
> Nice cleanup!
> 
> I don't know about the version bump, is it a breaking change compared to `yarn` `3.2`, or is it fully backwards compatible?
> 
> @pavera you may have some insights here?

I am new to yarn but I could not find any breaking changes from the [changelog](https://github.com/yarnpkg/berry/blob/master/CHANGELOG.md)

I saw some [API changes](https://github.com/yarnpkg/berry/blob/master/CHANGELOG.md#api-changes) but according to the release notes this should only impact people writing Yarn plugins

**Comment by deivid-rodriguez** [comment]
From the changelog you linked to, it's unclear what made it to 3.5.0 and what didn't, right? It does seem like a significantly bigger release than previous ones? Maybe we can first upgrade to 3.4.1 and once all good & stable, move on to 3.5.0?

**Comment by yeikel** [comment]
> From the changelog you linked to, it's unclear what made it to 3.5.0 and what didn't, right?

Could you please clarify what you mean here? 




> From the changelog you linked to, it's unclear what made it to 3.5.0 and what didn't, right? It does seem like a significantly bigger release than previous ones? Maybe we can first upgrade to 3.4.1 and once all good & stable, move on to 3.5.0?

That's fair. I created this https://github.com/yarnpkg/berry/issues/5369

And we can start with `3.4.1` for now

**Comment by deivid-rodriguez** [comment]
Oh, I see that you had actually understood me perfectly :). Thanks for asking upstream about this!

**Comment by yeikel** [comment]
> Oh, I see that you had actually understood me perfectly :). Thanks for asking upstream about this!

Thank you for your input

Is there anything else blocking this pull request?

**Review by deivid-rodriguez — APPROVED** [review]
No blockers from my side @yeikel!

I'll just wait a few more hours just in case @pavera has some input, and then merge away!

Thanks for you work!

**Review by jeffwidman — APPROVED** [review]

**Comment by deivid-rodriguez** [comment]
Turns out we're already using Yarn 3.5.0 for projects specifying `"packageManager": "yarn@3.5.0"` (though `corepack`) and nobody has reported any issues, so I think we can move on this upgrade!

**Pull request merged**